specifications/api/swagger.yaml

```diff
@@ -887,7 +897,8 @@ paths:
       operationId: listTransactions
       tags: ["Transactions"]
       summary: List
-      description: Priority [High]
+      description: |
+        <p align="right">status: <strong>not implemented</strong></p>
```

Comment:
To me, the phrase "not implemented" might be read as a deliberate decision to not implement something.

Perhaps we could write "not yet implemented" here?

Reply:
Hmmm... I don't think so :sweat_smile: ... If something wasn't going to be implemented, it wouldn't be in the API specification at all in a first place. Here the meaning is quite unambiguous.

Reply:
Fair enough. (My original thinking was prompted by the use of "isn't yet supported." elsewhere.)